R/model-tools.R

```diff
@@ -31,43 +31,39 @@
 #' enw_formula_as_data_list(prefix = "missing")
 enw_formula_as_data_list <- function(formula, prefix,
                                      drop_intercept = FALSE) {
-  paste_lab <- function(string, lab = prefix) {
-    paste0(lab, "_", string)
-  }
+  # nolint start: object_name_linter (because it produces false positives in within() calls)
+  data <- within(list(), {
```

Comment:
This could just be: 

```r
 list(
  stuff = stuff,
 stuff2 = ...,
)
```

Now that we aren't doing the stacked assignment thing + can drop the linting gates.

Reply:
dropped the lint gates but the list won't work like that on my machine (i probably don't understand your suggestion) so i left as is